fastai/model.py

```diff
@@ -95,7 +95,7 @@ def fit(model, data, epochs, opt, crit, metrics=None, callbacks=None, stepper=St
             for cb in callbacks: cb.on_batch_begin()
             loss = stepper.step(V(x),V(y), epoch)
             avg_loss = avg_loss * avg_mom + loss * (1-avg_mom)
-            debias_loss = avg_loss / (1 - avg_mom**batch_num)
+            debias_loss = float(avg_loss / (1 - avg_mom**batch_num))
```

Comment:
Might be a good time to ensure this does not result in a NaN, if the divisor is zero. Add a really small epsilon if the denominator is zero ??